lib/ioc-rc

```diff
@@ -555,7 +557,8 @@ __rc_jails () {
         local boot_order=$(sort -n $boot_list)
         local shutdown_order=$(sort -rn $boot_list)
     else
-        __info "no jails found with property boot=on, exiting.."
+        __info "no jails found with property boot=on or
+        with jail_hostid: $hostid on this host, exiting.."
```

Comment:
If I do that then the error on the console gets outlined.

This is the error the way I get it

root@node3:/usr/local/lib/iocage # /usr/local/etc/rc.d/iocage start
  INFO: no jails found with property boot=on or
           with jail_hostid: deffca19-ceff-e91c-ceff-e91cd3ff6a1a on
this host, exiting..

If I use 4 more spaces the console output looks like this

root@node3:/usr/local/lib/iocage # /usr/local/etc/rc.d/iocage start
  INFO: no jails found with property boot=on or
                with jail_hostid: deffca19-ceff-e91c-ceff-e91cd3ff6a1a
on this host, exiting..

I think the former output is nicer.

Op 15/09/15 om 00:10 schreef Brandon Schneider:
>
> In lib/ioc-rc
> <https://github.com/iocage/iocage/pull/222#discussion_r39453883>:
>
> > @@ -555,7 +557,8 @@ __rc_jails () {
> >          local boot_order=$(sort -n $boot_list)
> >          local shutdown_order=$(sort -rn $boot_list)
> >      else
> > -        __info "no jails found with property boot=on, exiting.."
> > +        __info "no jails found with property boot=on or
> > +        with jail_hostid: $hostid on this host, exiting.."
>
> Yes, the line that reads:
>
> |with jail_hostid: $hostid on this host, exiting.." |
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/iocage/iocage/pull/222/files#r39453883>.
>